docs/Home.md

```diff
@@ -1,3 +1,13 @@
-Welcome to the android-templates wiki! 
+Welcome to the android-templates wiki!
 
-More coming soon...
\ No newline at end of file
+The purpose of our android-templates repository is to quickly start a new project with all the **must-have** requirements. Other **nice-to-have** features need to be considered as an optional setting when starting a new project.
+
+Planning to add a new feature? Ask yourself the following questions to define if it's a **must-have** requirement:
+
+- Is the feature frequently used in other projects? (Ex: Glide, Retrofit, ...)
+- Does the feature increase productivity of the team? (Ex: Firebase, Proxyman, ...)
+
+Do _both_ apply to your feature? If not, we will define it as a **nice-to-have** feature:
+
+- Ex: [Jetpack Compose](https://developer.android.com/jetpack/compose) simplifies writing code for UI, but is not frequently used (yet)
```

Comment:
it doesn't simplify, it is "a different way to write UI programmatically". The way you interpreted it makes it sounds like a nice-to-have not by itself 😅

Reply:
> Jetpack Compose is Android's modern toolkit for building native UI. It simplifies and accelerates UI development on Android bringing your apps to life with less code, powerful tools, and intuitive Kotlin APIs

I got it from the official documentation actually 😅 But I'll find another example anyways, as in the future it will be used more frequently in our projects 🚀